nectar/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+-   New command to archive swaps: `nectar archive-swap <swap id>`.
+    The command should be done while nectar is stopped as it updates the database.
+    Archiving a swap stops nectar to resume its automated execution, `create-transaction` command can then be used to recover funds.
+
+### Changed
+
+-   **DB Change**: Add an "archived" field to stored swap data;
```

Comment:
This is not a separate feature, hence I would mention it twice.

What do you think about including a general notice at the beginning of the changelog that says: "This release includes changes to the DB schema, we recommend to backup your data directory before upgrading"?